recipe/meta.yaml

```diff
@@ -1,48 +1,28 @@
 {% set name = "anaconda-cloud-auth" %}
-{% set version = "0.7.2" %}
-{% set python_min = "3.8" %}
+{% set version = "0.8.2" %}
+{% set python_min = "3.9" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
-source:
-  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/anaconda_cloud_auth-{{ version }}.tar.gz
-  sha256: bdb49cdb5a22b5d0c1d825cf2b0529005628fa987b5a1466283d80722590044a
-
 build:
   noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
   number: 0
```

Comment:
Not sure if it helps but for conda-token and other metapackage I've seen. `noarch: generic` used

https://github.com/anaconda/conda-token/blob/main/conda.recipe%2Fmeta.yaml#L10-L12

Reply:
Oh! Forgot about that